docs/querying/multi-value-dimensions.md

```diff
@@ -25,7 +25,7 @@ title: "Multi-value dimensions"
 
 Apache Druid supports "multi-value" string dimensions. These are generated when an input field contains an
 array of values instead of a single value (e.g. JSON arrays, or a TSV field containing one or more `listDelimiter`
-characters).
+characters), by default they are sorted alphabetically, see [Dimension Objects](../ingestion/index.html#dimension-objects) for configuration.
```

Comment:
```suggestion
characters), by default Druid lists the values in alphabetical order, see [Dimension Objects](../ingestion/index.md#dimension-objects) for configuration.
```
Please use `.md` for links. Also, it's not clear to me what the related information is at the destination. But I'm a still fairly new to Druid.

Reply:
Thanks @techdocsmith, I've updated the changes. "Lists" is the wrong verb here, "ingests" would be more appropriate. The referred section is the above committed detail about multiValueHandling.

Reply:
Also `.md` should be `.html`, check the rest of the reference in the file. The markdown is compiled for the web version of the doc.

Reply:
@lkm , please see: https://github.com/apache/druid/pull/10528 Using .md is more flexible for viewing docs locally and for publishing to the web

Reply:
Thanks for the context @techdocsmith, I've changed it to .md